options_conda_build.py

```diff
@@ -1,47 +1,10 @@
 import os
 
-PREFIX = os.environ.get('PREFIX')
-SRC_DIR = os.environ.get('SRC_DIR')
 
-QTDIR = PREFIX
-QT4_BINPATH = PREFIX + "/bin"
-QT4_CPPPATH = PREFIX + "/include"
-QT4_LIBPATH = PREFIX + "/lib"
-QT4_FRAMEWORK = False 
-gl_include = PREFIX + "/include"
-gl_lib = PREFIX + "/lib"
-python_includes = PREFIX + "/include/python2.7"
-python_libpath = PREFIX + "/lib"
-boost_includes = PREFIX + "/include"
-boost_lib = PREFIX + "/lib"
-openalea_lib = PREFIX + "/lib"
-openalea_includes = PREFIX + "/include"
-build_prefix = SRC_DIR + "/build-scons"
-build_includedir = PREFIX + "/include"
-build_libdir = PREFIX + "/lib"
-build_bindir = PREFIX + "/bin"
-WITH_CGAL = True 
-cgal_includes = PREFIX + "/include"
-cgal_libpath = PREFIX + "/lib"
-bison_bin = PREFIX + "/bin"
-flex_bin = PREFIX + "/bin"
-flex_libpath = PREFIX + "/lib"
-flex_include = PREFIX + "/include"
-pthread_includes = PREFIX + "/include"
-pthread_lib = PREFIX + "/lib"
-WITH_QHULL_2011 = True 
-qhull_includes = PREFIX + "/include"
-qhull_lib = PREFIX + "/lib"
-gmp_includes = PREFIX + "/include"
-gmp_libpath = PREFIX + "/lib"
-WITH_GMP = True 
-mpfr_includes = PREFIX + "/include"
-mpfr_libpath = PREFIX + "/lib"
-WITH_MPFR=True
-ann_includes = PREFIX + "/include"
-ann_libpath = PREFIX + "/lib"
-WITH_LAPACK = False 
-WITH_EIGEN = True 
-eigen_includes = PREFIX + "/include/eigen3"
-EXTRA_CXXFLAGS = "-DCGAL_CFG_NO_CPP0X_VARIADIC_TEMPLATES -DWITH_QHULL_2011" 
-EXTRA_LIBS = "boost_system boost_thread png" 
+EXTRA_CXXFLAGS="-DCGAL_CFG_NO_CPP0X_VARIADIC_TEMPLATES"
+
+if 'CPU_COUNT' in os.environ:
+    num_jobs = os.environ['CPU_COUNT']
+
+
+EXTRA_LIBS = "boost_system png" 
```

Comment:
Well done, guy!!! You successfully remove quite everything from option.py.
Perhaps we need to add boost_system deps with boost in sconsx and png with??? Qt perhaps?

Reply:
agreed